src/dashboard/src/components/administration/views_processing.py

```diff
@@ -64,6 +72,57 @@ def write_to_file(self, file_path):
         file = open(file_path, 'w')
         file.write(etree.tostring(self.xml, pretty_print=True))
 
+
+def save__normalize(name, value, choice_uuid, do_not_lookup, xml_choices, default_processing_func):
+    """
+    There are two "Normalize" chain links with choices "Do not normalize", refs #10216.
+    """
+    if value != 'Do not normalize':
```

Comment:
The `<appliesTo>` is different for the other nodes (preservation, access, service for access), so they also need the special handling to work for transfers with `/access`.  Might be easiest to do the special handling for all `if name == 'normalize'`?